editor.html

```diff
@@ -15183,5 +15183,6 @@ <h2 id="flashing-text">Flashing micro:bit</h2>
 :0400000500018E2147
 :00000001FF
 </div>
+</main>
```

Comment:
Did we have a missing closing `div` all this time?

Reply:
It appears so at this point. Not sure historically.

Reply:
In that case let's close `</main>` before the XML with the Blockly toolbox and the hidden div with the MicroPython firmware.

Reply:
Done

Reply:
Regression as well.